lib/mini_profiler/storage/memory_store.rb

```diff
@@ -70,7 +70,7 @@ def initialize_cleanup_thread(args={})
         cleanup_cycle    = args.fetch(:cleanup_cycle)    { CLEANUP_CYCLE }
         t = CacheCleanupThread.new(cleanup_interval, cleanup_cycle, self) do |t|
           until Thread.current[:should_exit] do
-            self.sleepy_run
+            CacheCleanupThread.current.sleepy_run
```

Comment:
wait, should this not be `t.sleepy_run` also seems a bit confusing that `t` is being shadowed.

Reply:
No idea why `t`is being shadowed.

I tried changing it so that `t` is not being shadowed and referenced `t.sleepy_run` using the block scope(?), which results in the error:
`lib/mini_profiler/storage/memory_store.rb:73:in 'block in initialize_cleanup_thread': undefined method 'sleepy_run' for 10:Fixnum (NoMethodError)`

Any ideas? Using `CacheCleanupThread.current.sleepy_run` was the least invasive way to fix the problem quickly.

Reply:
Yeah I will merge for now, thanks!

Reply:
Thanks @SamSaffron.
Sorry, I did not have the time to thoroughly debug the problem.